client/views/test.js

```diff
@@ -0,0 +1,73 @@
+/*
+Need:
+	How to log in - add isAdmin later
+	Modale/stylish popup for adding event
+	Drop down/window for displaying event info
+*/
+if (Meteor.isClient) {
```

Comment:
We should be the client -- probably remove this check for consistency across js view files :)